rendering/src/camera.rs

```diff
@@ -0,0 +1,55 @@
+use std::mem;
+
+use common::{Mat4, Vec3};
+
+pub struct Camera {
```

Comment:
Possibly put this in common?

Reply:
I definitly think that we should have some kind of CameraComponent that can be added to an entity (or possibly multiple?) and then the renderer would query for those to know where to render from.

Then the CameraComponent would probably be in common i guess

But of course the renderer would have to know about CameraComponents